docs/source/concepts/cl_columns.rst

```diff
@@ -46,12 +64,54 @@ Contains a boolean value, with ``True`` indicating that the sample is likely to
 .. _suggested_label:
 ``suggested_label``
 ---------------
-Contains the suggested label for the sample. If the sample is not a label issue (``is_label_issue == False``), the suggested label will be empty. For samples with label issues, the suggested label is computed by Cleanlab studio.
+Contains the suggested label for the sample. If the sample is not a label issue (``is_label_issue`` marked as ``False``), the suggested label will be empty. For samples with label issues, the suggested label is computed by Cleanlab studio.
 
 Outliers
 ========
 
 .. _is_outlier:
 ``is_outlier``
 -------
-Contains a boolean value, with ``True`` indicating that the sample is likely to be an outlier.
\ No newline at end of file
+Contains a boolean value, with ``True`` indicating that the sample is likely to be an outlier.
+
+Ambiguous
+=========
+
+.. _is_ambiguous:
+``is_ambiguous``
+----------
+Contains a boolean value, with ``True`` indicating that the sample is likely to be ambiguous. Ambiguous samples are those that do not obviously belong to a single class.
+
+High Confidence
+===============
+
+.. _is_high_confidence:
+``is_high_confidence``
+---------------
+Contains a boolean value, with ``True`` indicating high confidence, which means that the given label of the sample is likely to be correct, so the sample can be safely used in downstream tasks.
+
+Uncertain
+=========
+
+.. _is_uncertain:
+``is_uncertain``
+-----------
+Contains a boolean value, with ``True`` indicating uncertain. Uncertain data specifically means data that are likely to be mislabeled (they will have ``is_label_issue`` marked ``True``), but that Cleanlab is uncertain about what their correct label should be. Such data would benefit from human review.
```

Comment:
Is this just the negation of `is_high_confidence`? If yes, remove it; if not, would be nice to briefly explain how the two are different.

Reply:
they are related but not quite negations:
- `is_high_confidence` (which will become `is_given_label_correct` is where P[suggested] is high and label quality is high
- `is_uncertain` is where sample is marked with label issue, and P[suggested] is low

But as per discussion we will not include `is_uncertain` for now because of its similarity with ambiguous (see https://cleanlabinc.slack.com/archives/C02SN6TFSF8/p1686948331146719)